app/views/verify/authentications/failed.html.erb

```diff
@@ -2,22 +2,35 @@
   <div class="govuk-grid-column-two-thirds">
 
     <h1 class="govuk-heading-xl">
-      Your identity could not be verified
+      We could not verify your identity
     </h1>
 
     <p class="govuk-body">
-      There are a few reasons why this might happen, including:
+      This might be because:
     </p>
 
     <ul class="govuk-list govuk-list--bullet">
       <li>
-        your answer to a question doesn't match the records held by the Verify provider
+        the company you chose does not have enough information about you
       </li>
       <li>
-        the Verify provider that you chose does not have access to enough information about you
+        the company’s records are not up to date
       </li>
       <li>
-        you've recently had a change in circumstances, and they are not up to date online
+        your answers did not match the company’s records
+      </li>
+    </ul>
+
+    <h2 class="govuk-heading-m">
+      Next steps
+    </h2>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        check your answers are correct
+      </li>
+      <li>
+        try using a different company
```

Comment:
`try using a different company for verification` maybe?

Reply:
If we were going to do this then I would probably go for 'try choosing a different company in GOV.UK Verify'

Reply:
@PatrickTaylorDfE do you have a preference either way?

Reply:
My preference would be to leave as is.

Reply:
Cool, let's leave it then. I'm probably overthinking it 😄